dict/P/pandunia.tsv

```diff
@@ -663,6 +668,7 @@ PWN:europium.n.01		europium	europ·ium	eng:europium, fra:europium, spa:europio,
 PWN:even.r.01		hata	hata	ara: حَتَّى (ḥatta), fas:حَتّی, tur:hau:hatta, swa:hata, ful:haa, spa:hasta, por:até
 PWN:example.n.01		misal	misal	ara:مِثَال, may:misal, tur:misal, fas:مِثال, hin:मिसाल
 PWN:excellent.s.01		top klasik	top klas·ik	
+PWN:excessively.r.01		tu	tu	eng:too

```

Comment:
Unfortunately _tu_ already has a meaning, 'you'.

Reply:
I don't know how I forgot that ''tu'' was a word.  What about ''tai'', then?